config/default/dashboard.dashboard.dashboard.yml

```diff
@@ -0,0 +1,127 @@
+uuid: 9e622af8-2093-42ab-a7fe-0f1408c29a7b
+langcode: en
+status: true
+dependencies:
+  config:
+    - views.view.duplicate_people
+    - views.view.editoria11y_results
+    - views.view.my_latest_edits
+    - views.view.new_default_image_alt_text
+    - views.view.site_dashboard_active_site_editors
+  module:
+    - hs_dashboard
+    - layout_builder
+    - layout_discovery
+    - views
+id: dashboard
+label: Dashboard
```

Comment:
There may be more dashboards in the future? Calling this "Home Dashboard" or "Main Dashboard" could help future-proof.